qemu/tests/cfg/qsd_live_snapshot.cfg

```diff
@@ -0,0 +1,37 @@
+- qsd_live_snapshot:
+    type = qsd_live_snapshot
+    start_vm = no
+    images + = "stg1"
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    base_tag = "stg1"
+    snapshot_tag = sn1
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+    device = "fmt_stg1"
+    format = qcow2
+    rebase_mode = unsafe
+    only Linux
+    !Host_RHEL.m7:
+        node = "fmt_stg1"
+        overlay = "drive_sn1"
+        qemu_force_use_drive_expression = no
+
+    # QSD attributes
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1"
+    # Image attributes
+    image_name_stg1 = images/stg1
+    image_size_stg1 = 2G
+    image_size_sn1 = ${image_size_stg1}
+    drive_format_stg1 = vhost-user-blk-pci
+    image_vubp_props_stg1=  {"num-queues":1}
```

Comment:
suggest to add these two params for sn1, refer to the reason below

Reply:
image_vubp_props

Reply:
Updated as your suggestion, thanks